app/dragontts/app/providers/gemini.py

```
@@ -0,0 +1,291 @@
+            if _cancel is not None:
+                try:
+                    _cancel()
+                except Exception:
+                    pass
+
+    async def aclose(self) -> None:
+        """Release the cached gRPC channel (built lazily on first use)."""
+        if self._client is not None:
+            try:
+                await self._client.transport.grpc_channel.close()  # type: ignore[union-attr]
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)